pulpcore/pulpcore/app/models/task.py

```diff
@@ -36,7 +36,7 @@ class ReservedResource(Model):
     resource = models.TextField(unique=True, blank=False)
 
     tasks = models.ManyToManyField("Task", related_name="reserved_resources")
-    worker = models.ForeignKey("Worker", on_delete=models.CASCADE, related_name="reservations")
+    worker = models.ForeignKey("Worker", related_name="reservations", on_delete=models.CASCADE)
```

Comment:
Doesn't the worker crash recovery rely on the presence of old `ReservedResource` records in order to know which tasks to clean up `created_resources` with `complete=False`? I recognize you are only moving this, but I'm wondering if its correct to begin with.

Reply:
I don't know... "practically" speaking this is fine since we don't delete worker models any more.. but I think we should probably audit the models before the beta to make sure nothing like that is slipping through the cracks.

Reply:
I appreciate Django 2's decision to make ```on_delete``` explicit for that reason

Reply:
That sounds fine. I agree we need to look over all the `on_delete()` relationships soon.